_examples/12_simple_validation/proto/federation/federation.proto

```diff
@@ -39,7 +33,23 @@ message GetPostResponse {
       {
         error: {
           code: FAILED_PRECONDITION,
-          rule: "post.title == 'some-title'",
+          details: {
+            rule: "$.id == 'correct-id'",
+            precondition_failures: {
+              violations: [
+                {
+                  type: "'type1'"
```

Comment:
It would indeed be more user-friendly if they didn't have to wrap it with `''`. How about we first parse it as CEL; if that returns an error, we treat the field as a string. We may add a new field to the CELValue to indicate the field could be either CEL or plane string.